packages/cozy-client/src/models/file.js

```diff
@@ -665,50 +665,55 @@ export const isExpiring = file => {
 }
 
 /**
- * @param {IOCozyFile} file - An CozyFile
- * @param {string} dateLabel - Label of date
- * @returns {Date} Normalize expiration date
- */
-export const computeNormalizeExpirationDate = (file, dateLabel) => {
-  if (file.metadata[dateLabel]) {
-    if (dateLabel === 'referencedDate') {
-      return add(new Date(file.metadata[dateLabel] ?? file.created_at), {
-        days: 365
-      })
-    }
-    return new Date(file.metadata[dateLabel])
+ * @param {IOCozyFile} file - io.cozy.files document
+ * @returns {Date | null} Expiration date
+ */
+export const computeExpirationDate = file => {
+  if (isExpiringFrenchNationalIdCard(file) || isExpiringResidencePermit(file)) {
+    const expirationDate = get(file, 'metadata.expirationDate', null)
+    return new Date(expirationDate)
   }
+  if (isExpiringPersonalSportingLicense(file)) {
+    const referencedDate = get(file, 'metadata.referencedDate', null)
+    const created_at = get(file, 'created_at', null)
+    return add(new Date(referencedDate ?? created_at), {
+      days: 365
```

Comment:
this 365 should be added to the `config` we talked early.

Reply:
Isolated as a `PERSONAL_SPORTING_LICENCE_PERIOD_DAYS` constant for now, i'll try to see how i can integrate that to a config file.